service/src/main/java/org/folio/rest/workflow/config/KafkaProducerConfig.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rest.workflow.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.folio.spring.messaging.model.Event;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+import org.springframework.kafka.support.serializer.JsonSerializer;
+
+@Configuration
+@Profile({ "messaging", "!test" })
+public class KafkaProducerConfig {
+
+  @Value(value = "${spring.kafka.bootstrap-servers}")
```

Comment:
It would be great if the READMEs where updated.

It is quite lacking as it is but given that Kafka has just been added, I think it would be great to have this added now.
(This specific documentation request should not block the approval of this PR I just wanted to mention this.)

Reply:
There are a lot of default properties in application.yaml that are not documented nor ported to the env section of the module descriptor.